### PR TITLE
Link the User ID to the Edit Profile page

### PR DESCRIPTION
**Pull request by zackkatz**
I always want to see the user profile, not just the customer details.
Alternately, other user details like the username would be helpful.

### CODE REVIEW

**Comment by zackkatz** [comment]
My current flow: 

* Go to the customer
* Copy the User ID
* Go to the Users page
* Click on any user
* Change the ID parameter in the URL
* Access the user profile

I do this a *lot*, and having this link would fix it.  The CSS should probably be updated, but I didn't want to mess with your process on how to name/implement.

**Comment by shazahm1** [comment]
+1

File: includes/admin/customers/customers.php [hunk 1]
```diff
@@ -283,8 +283,8 @@ function edd_customers_view( $customer ) {
 
 					<span class="customer-user-id info-item editable">
 						<?php _e( 'User ID', 'edd' ); ?>:&nbsp;
-						<?php if( intval( $customer->user_id ) > 0 ) : ?>
-							<span data-key="user_id"><?php echo $customer->user_id; ?></span>
+						<?php if( intval( $customer->user_id ) > 0 ) : die(); ?>
```
Comment (pippinsplugins) [inline includes/admin/customers/customers.php]: Dying here is not right. We still need it to continue with the rest of the page.

Comment (pippinsplugins) [inline includes/admin/customers/customers.php]: @zackkatz

Comment (zackkatz) [inline includes/admin/customers/customers.php]: Shoot, sorry @pippinsplugins, that was silly testing.

Comment (pippinsplugins) [inline includes/admin/customers/customers.php]: Could you remove it so we can merge this in?

**Comment by zackkatz** [comment]
@pippinsplugins done.

**Pull request merged**